lib/zlib.js

```diff
@@ -229,9 +229,15 @@ class Zlib extends Transform {
     var strategy = constants.Z_DEFAULT_STRATEGY;
     if (typeof opts.strategy === 'number') strategy = opts.strategy;
 
-    this._handle.init(opts.windowBits || constants.Z_DEFAULT_WINDOWBITS,
+    var windowBits = constants.Z_DEFAULT_WINDOWBITS;
```

Comment:
They could, and I'd like them to be, but I just wrote these in a style consistent with the code above. How about me addressing this comment in a follow-up PR, together with some more refactoring like replacing most `var`s with `let`s and `const`s?